doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,162 @@
+# Owner References for Existing Resources
+
+Owner references are automatically injected *only during creation of
+resources*. Enabling owner reference injection *will not update objects*
+created while [owner reference injection is
+disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+.
+
+This guide will demonstrate how to retroactively set owner references
+for existing resources.
+
+### Owner References and Annotations
+
```

Comment:
I  think would be easier to understand here if we had 2 subsections something as: 

```suggestion
### For objects in the same namespace of the Owner(CRD)
### For objects which are NOT in the same namespace of the Owner(CRD)
```